libclambcc/ClamBCRemoveUndefs/ClamBCRemoveUndefs.cpp

```diff
@@ -240,13 +272,40 @@ class ClamBCRemoveUndefs : public ModulePass
             delLst[i]->eraseFromParent();
         }
 
-        return bChanged;
+        errs() << "<" << __LINE__ << ">" << "99\% sure this is wrong, since this pass does NOT preserve analyses<END>\n";
+        return PreservedAnalyses::all();
     }
 }; // end of struct ClamBCRemoveUndefs
 
 } // end of anonymous namespace
 
+#if 0
```

Comment:
Is this `#if` temporary for the sake of testing?  Do you need to keep the old way?